Update issue template to make clear that plugins can be submitted as Beta (#105)
mattermost/mattermost-marketplace: .github/ISSUE_TEMPLATE/add_plugin.md

## Checklist
<!--
Go through this checklist and confirm every item.

> It's fine, if your plugin doesn't fullfil every item i.e. isn't production ready yet. You can still submit it!

Reviewer: ```suggestion
It's fine if your plugin doesn't fulfill every item (e.g. it isn't production ready yet). You can still submit it!
```

> You can still do code changes while the plugin is in review and fix issues on the fly.

Reviewer: ```suggestion
You can also still do code changes while the plugin is in review and fix issues on the fly.
```

> Even if your plugin isn't production ready, it might still be added to the Marketplace as "Beta".

Reviewer: ```suggestion
Even if your plugin isn't production-ready, it can still be added to the Marketplace as "Beta".
```

Author: ```suggestion
Even if your plugin isn't production-ready, it might still be added to the Marketplace as "Beta".
```

> Even if your plugin isn't production ready, it might still still be added to the Marketplace as "Beta".

Reviewer 2: "still still"

See https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/#beta-plugins for more details.

> If your plugin isn't production ready, please leave a comment stating if you plan to fulfill the whole checklist or intending to submit a "Beta" plugin.

Reviewer: ```suggestion
If your plugin isn't production ready, please leave a comment stating whether you plan to fulfill the whole checklist or submit a "Beta" plugin.
```
